projects/nae-example-app/src/app/doc/tabbed-case-view/tabbed-task-view/tabbed-task-view.component.ts

```diff
@@ -16,9 +16,12 @@ import {
     tabbedAllowedNetsServiceFactory,
     tabbedTaskViewConfigurationFactory,
     NAE_TASK_VIEW_CONFIGURATION,
-    ChangedFieldsService, NAE_ASYNC_RENDERING_CONFIGURATION
+    ChangedFieldsService, NAE_ASYNC_RENDERING_CONFIGURATION, TaskEventOutcome
 } from '@netgrif/components-core';
 import {HeaderComponent} from '@netgrif/components';
+import {
+    TaskEventNotification
+} from '../../../../../../netgrif-components-core/src/lib/task-content/model/task-event-notification';
```

Comment:
Shouldn't this be also `@netgrif/` import like the ones above?

Reply:
thx  bad  import

Reply:
fix :)  delete  import  
it was used for the test